qiskit/execute.py

```diff
@@ -69,8 +69,8 @@ def execute(experiments, backend,
 
         backend_properties (BackendProperties):
             Properties returned by a backend, including information on gate
-            errors, readout errors, qubit coherence times, etc. For a backend
-            that provides this information, it can be obtained with:
+            errors, readout errors, qubit coherence times, etc. Find a backend
+            that provides this information with:
```

Comment:
I think it is correct originally, but it was very wordy.  If I've changed the meaning when I made it more concise, let me know and I'll find a better way.  Thank you

Reply:
:+1: This is okay. I think the problem I was seeing is that this docstring describes what a `BackendProperties` object is and how to obtain it, instead of describing how the `backend_properties` argument impacts the `execute` function, but that's a separate issue.